### DISCUSSION TITLE
marketcollector error

### DISCUSSION THREAD

**Discussion by BlackTail13** [cyberjunky/3commas-cyber-bots#178] [Q&A]
Hi ... when I try running the marketcollector I get the error like this : 
 File "/usr/lib/python3.9/configparser.py", line 789, in get
    value = d[option]
  File "/usr/lib/python3.9/collections/__init__.py", line 941, in __getitem__
    return self.__missing__(key)            # support subclasses that define __missing__
  File "/usr/lib/python3.9/collections/__init__.py", line 933, in __missing__
    raise KeyError(key)
KeyError: 'timeinterval'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/lithium/3commas-cyber-bots/./marketcollector.py", line 919, in <module>
    sectiontimeinterval = int(config.get(section, "timeinterval"))
  File "/usr/lib/python3.9/configparser.py", line 792, in get
    raise NoOptionError(option, section)
configparser.NoOptionError: No option 'timeinterval' in section: 'altrank_default'

**Comment by amargedon** [comment]
Would have seen this post earlier when it was created as an issue (which it is), however, you don't have the mandatory 'timeinterval' property in the 'altrank-default' section. Add this property with a proper value and you should be good to go (also check the other sections you have in the .ini file).

**Reply by BlackTail13** [reply]
I already try but still same problem, because I'm blind in code just follow the instruction hehe ... btw thank friend for you reply

**Reply by amargedon** [reply]
Do you use the altrank data in your bots? Otherwise you could also delete this section (not a real solve and a workaround, but yeah).

**Reply by BlackTail13** [reply]
I'm not use altrank data, I try to understand what you mean for delete that section n try again

**Reply by amargedon** [reply]
Delete the section means remove the [altrank_] part and the lines below until the next section.
